app/src/main/res/layout/item_event.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+  <android.support.v7.widget.CardView
+      xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
set your android studio to 2space tab. Please use tab. Do not use spaces.
Indentations are good for reading code. 
<android